prisma/schema.prisma

```diff
@@ -9,14 +9,77 @@ datasource db {
 
 model User {
   id       String     @id @default(auto()) @map("_id") @db.ObjectId
-  email    String     @unique
-  name     String?
-  activity Activity[]
+  name     String
+  email    String     
+  password String
+  wishList String[]
+  activity Activity[] @relation("userActivity")
+  venueId Venue? @relation("userVenue")
+  createdAt DateTime @default(now())
+  updatedAt DateTime @updatedAt
 }
 
 model Activity {
   id     String @id @default(auto()) @map("_id") @db.ObjectId
   name   String
-  user   User   @relation(fields: [userId], references: [id])
+  description String
+  image String
+  venueId Venue? @relation("activityVenue")
+  duration Int
+  category Category @default(Indor)
+  numPlayers Int
+  minPlayers Int
+  maxPlayers Int
+  difficultRating DifficultyRating @default(Normal)
+  likeReview Float
+  user   User?   @relation("userActivity", fields: [userId], references: [id])
   userId String @db.ObjectId
+  location Location? @relation("venueLocation")
+  createdAt DateTime @default(now())
+  updatedAt DateTime @updatedAt
+}
+
+model Venue {
+  id String @id @default(auto()) @map("_id") @db.ObjectId
+  name String @unique
+  email String
+  description String
+  reviews String
+  logo String
+  mainPhoto String
+  bannerPhoto String
+  userVenue User? @relation("userVenue", fields: [userVenueId], references: [id])
+  userVenueId String @db.ObjectId @unique
```

Comment:
On model Venue you use `userId`, on model Activity , you use `userVenueId` if same field, choose a name and keep it.

We can discuss more this relationship.